marklogic-data-hub-central/ui/src/components/entities/matching/matching-step-detail/matching-step-detail.module.scss

```diff
@@ -157,27 +157,42 @@
 .testMatch {
     padding-left: 44px;
```

Comment:
Might want to adjust the padding left to 34px. It looks a bit inconsistent as compared to other sections in match step detail page. See the screenshot below.
![image](https://user-images.githubusercontent.com/19846034/118425978-34ed4980-b67f-11eb-8b5d-4a4e52d47179.png)

Reply:
Fixed both issues and is ready for review